src/components/MessagesList/MessagesList.vue

```diff
@@ -957,8 +957,7 @@ export default {
 					this.$refs.scroller.scrollTop += this.$refs.scroller.offsetHeight / 4
 				}
 				if (highlightAnimation) {
-					element.focus()
-					element.highlightAnimation()
+					element.dispatchEvent(new CustomEvent('highlight'))
```

Comment:
should be added to the `emits` on the vue component?

Reply:
I doubt that, as it's not a Vue event, but a native JS